python/cuml/cuml_accel_tests/conftest.py

```diff
@@ -12,10 +12,5 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from cuml.accel import install
-
-# Install the accelerator
-install()
```

Comment:
The `-p cuml.accel` CLI for pytest should do this already

Reply:
Yes, I was also wondering why we did both. Then this confirms that the issue with `pytest-xdist` is that we should not use the `pytest_load_initial_conftests` function to patch sklearn's `all_estimator` function.